Assets/JsonDotNet/Assemblies/Standalone/Newtonsoft.Json.dll.meta

```diff
@@ -0,0 +1,75 @@
+fileFormatVersion: 2
```

Comment:
We should NOT be using Newtonsoft due to it's compatibility issues on platforms.
This should be replaces / updated to use the Unity JSON implementation
https://docs.unity3d.com/Manual/JSONSerialization.html

Reply:
@StephenHodgson and @chrisfromwork have brought this up and I've agreed that swapping it out is best. Once we have time to stress test it with state data it'll be removed.